astroquery/heasarc/__init__.py

```diff
@@ -17,13 +17,13 @@ class Conf(_config.ConfigNamespace):
     """
     Configuration parameters for `astroquery.heasarc`.
     """
-    server = _config.ConfigItem(
-        ['http://heasarc.gsfc.nasa.gov/cgi-bin/W3Browse/w3query_noredir.pl'],
-        'Name of the HEASARC server to use.')
+    #server = _config.ConfigItem(
+    #    ['http://heasarc.gsfc.nasa.gov/cgi-bin/W3Browse/w3query_noredir.pl'],
+    #    'Name of the HEASARC server to use.')
 
     # The above server does not work for querying available missions.
     # The following server does.
-    mission_server = _config.ConfigItem(
```

Comment:
The usage of `mission_server` needs to be changed to `server` in the code, too.

Reply:
The `mission_server` is used in `query_mission_list`, it returns a table [https://heasarc.gsfc.nasa.gov/db-perl/W3Browse/w3query.pl?tablehead=BATCHRETRIEVALCATALOG_2.0%20xxx](url). I did not figure out how to read this table.

Reply:
OK, then should we switch the uses of `mission_server` to `server` in the core code?